### PR TITLE
Campaign removal

### PR DESCRIPTION
**Pull request by benghaziboy**
@MichaelButkovic @mparent61 

removing dependencies on campaign, but not quite deleting the app yet for migration purposes.

### CODE REVIEW

File: bulbs/campaigns/models.py [hunk 1]
```diff
@@ -44,19 +44,18 @@ def pixel_dict(self):
...
-    DETAIL = 1
-    PIXEL_TYPES = (
-        (LISTING, 'Listing'),
-        (DETAIL, 'Detail'),
-    )
-
-    campaign = models.ForeignKey(Campaign, related_name='pixels')
-    url = models.URLField()
-    pixel_type = models.IntegerField(choices=PIXEL_TYPES, default=LISTING)
+# class CampaignPixel(models.Model):
```
Comment (mparent61) [inline bulbs/campaigns/models.py]: Any reason not to just delete this entirely?

Comment (benghaziboy) [inline bulbs/campaigns/models.py]: nope good catch

File: bulbs/special_coverage/models.py [hunk 2]
```diff
@@ -107,15 +104,12 @@ def _save_percolator(self):
...
         # support missing fields, so always need to include
-        if self.campaign:
-            q["start_date"] = self.campaign.start_date
-            q["end_date"] = self.campaign.end_date
-        else:
-            q["start_date"] = self.start_date
-            q["end_date"] = self.end_date
+
+        q["start_date"] = self.start_date
+        q["end_date"] = self.end_date
```
Comment (mparent61) [inline bulbs/special_coverage/models.py]: Are we ever going to sync/unify SpecialCoverage.start_date and Tunic Campaign.start_date?

Comment (benghaziboy) [inline bulbs/special_coverage/models.py]: I believe @spra85 said for the time being we are just going to rely on them enforcing the start/end dates on the special coverage object iself

Comment (spra85) [inline bulbs/special_coverage/models.py]: Correct.  I think that's easiest for now.

**Comment by mparent61** [comment]
LGTM

**Comment by benghaziboy** [comment]
:angel:

**Comment by MichaelButkovic** [comment]
👍

**Pull request merged**